examples/operator/101_initial_cluster.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     vtbackup: vitess/lite:latest
```

Comment:
Testing this branch and the vtctld image ends up in a crashloop backoff with the following error: 
unknown flag: --enable_realtime_stats

Reply:
That will only happen if you are using an older version of `operator` and not the latest version.